typhon/signal.py

```diff
@@ -147,7 +152,7 @@ def add_signal(self, signal, name):
         """
         logger.debug("Adding signal %s", name)
         # Create the read-only signal
-        read = signal_widget(signal, read_only=True)
+        read = signal_widget(signal, read_only=True, tooltip=tooltip)
```

Comment:
So this goes on the read only widget? I kind of imagined it would go on the `QLabel`. That is created in `_add_row`

Reply:
Good call. 👍